TLM/TLM/Util/VersionUtil.cs

```diff
@@ -128,27 +128,24 @@ public static class VersionUtil {
                 Version current = CurrentGameVersion.Take(VERSION_COMPONENTS_COUNT);
                 Version expected = ExpectedGameVersion.Take(VERSION_COMPONENTS_COUNT);
 
-                if (current < expected) {
-                    // game too old
+                if (current != expected) {
```

Comment:
At first I thought it's a typo, but as the new update brings breaking changes it might be done this way.

Reply:
See comment above `VERSION_COMPONENTS_COUNT` in code - CO update that part of the version whenever there are likely to be mod-breaking changes.

I can change it back to `<` | `>` if desired.

Reply:
> I can change it back to < | > if desired.

No need, it's OK.